api/src/main/java/com/ford/labs/retroquest/actionitem/CreateActionItemRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.ford.labs.retroquest.actionitem;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+import java.sql.Date;
+
+@Value
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CreateActionItemRequest {
+    String task;
+    boolean completed;
+    String assignee;
+    @EqualsAndHashCode.Exclude
```

Comment:
Not something we need to fix with this PR, but would be curious to have a conversation around why we'd want to exclude this from equals and hashcode.